maxim/logger/together/helpers.py

```
@@ -0,0 +1,90 @@
+            
+            if is_local_trace and trace is not None:
+                trace.set_output(accumulated_content)
+                trace.end()
+                
+        except Exception as e:
+            scribe().warning(
+                f"[MaximSDK][TogetherInstrumentation] Error in streaming generation: {e}",
+            )
+
+    @staticmethod
+    async def async_stream_helper(
```

Ruff (0.11.9)

Missing return type annotation for staticmethod `async_stream_helper`

(ANN205)